auditbeat/docs/fields.asciidoc

```diff
@@ -43,11 +43,19 @@ The file metricset generates events when a file changes on disk.
 [float]
 === `audit.file.path`
 
-type: keyword
+type: text
 
 The path to the file.
 
 [float]
+=== `audit.file.path.raw`
```

Comment:
Do we want these to show up in the docs? I added code here to do it, but I'm on the fence.

Reply:
I am OK to have it in the doc since its useful for aggregation.
Even if we don't display it in our doc it will still be accessible in the UI :)